### DISCUSSION TITLE
There are some optimizations under the ArchLinux KDE 5.27.4 Wayland protocol

### DISCUSSION THREAD

**Discussion by Silentdoer** [rustdesk/rustdesk#4049] [Ideas]
control side is windows 10, The controlled side is archlinux kde plasma 5.27.4 wayland.

rustdesk version is nightly

1.When the KDE display goes black, the control side rustdesk can connect to rustdesk under archlinux KDE, but it keeps getting screen information,The ArchLinux KDE screen must be activated manually with a mouse, keyboard, etc., to complete the remote connection

2.The control side rustdesk enabled the remote system of the lock screen after closing the connection, but it did not take effect after closing the connection

3.After the control side rustdesk connects to the ArchLinux KDE computer, and then enters the password to log in, the ArchLinux side will directly display the interface after logging in, which is not safe, because if there is someone near the ArchLinux computer, he can directly see and operate the computer

**Comment by rustdesk** [comment]
1, https://github.com/rustdesk/rustdesk/issues/230
2, @fufesou 
3, No plan for Linux yet.
